test/fixture/desc/p2pkh_desc_w_pubkeyhash_type.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Let's not check in json file, unless absolutely necessary.

Reply:
It's needed in the test for PubKeyHash. Would it be better to compile the sCrypt source file itself in the test?